packages/design-system/src/mixins/locale.ts

```diff
@@ -3,7 +3,7 @@ import { t } from '../locale';
 export default {
 	methods: {
 		t(path: string, ...args: string[]) {
-			return t.call(this, path, ...args);
```

Comment:
how does this not break [this line](https://github.com/n8n-io/n8n/blob/enforce-strict-for-design-system/packages/design-system/src/components/N8nFormInput/FormInput.vue#L243)?

Reply:
Misread `...args: unknown[]` in `N8nLocale` as belonging to `N8nLocaleTranslateFn`. I'll recheck.